rust_snuba/rust_arroyo/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
```

Comment:
Do you plan to actually publish a crate from rust_arroyo and import that eventually or do you plan to vendor rust_arroyo into snuba permanently ?

The first is better.

Reply:
I think the first is better in the long term and the second is better in the short term. I would hold off publishing a crate until we have a much better idea of what that library would look like. Right now we are unsure and prioritising for faster iteration.